server/selection.go

```diff
@@ -110,12 +153,14 @@ func NewMinLSSelector(stakeRdr stakeReader, minLS float64, selectionAlgorithm co
 	heap.Init(knownSessions)
 
 	return &MinLSSelector{
-		knownSessions:      knownSessions,
-		stakeRdr:           stakeRdr,
-		selectionAlgorithm: selectionAlgorithm,
-		perfScore:          perfScore,
-		capabilities:       capabilities,
-		minLS:              minLS,
+		knownSessions: knownSessions,
+		minLS:         minLS,
+		Selector: Selector{
```

Comment:
Should we use the constructor?

Reply:
We could, but it's just the embedded struct, so I'll probably keep it as it is.